Sources/CommandReport.swift

```diff
@@ -43,22 +33,22 @@ class CommandReport: Command {
             return
         }
         
+        if reporter == nil {
+            reply("Waiting for the filter to load...")
+            repeat {
+                sleep(1)
+            } while reporter == nil
+        }
+        
         guard let question = try apiClient.fetchQuestion(questionID).items?.first else {
             reply("Could not fetch the question!")
             return
         }
         
-        if usageIndex == 0
-        {
-            var newTitle = "\(question.title ?? "<no title>")"
-            
-            newTitle = newTitle.replacingOccurrences(of: "[", with: "\\[")
-            newTitle = newTitle.replacingOccurrences(of: "]", with: "\\]")
-            
-            let messagePost = "[ [\(botName)](\(stackAppsLink)) ] " +
-            "[tag:\(tags(for: question).first ?? "<unknown tag>")] Manually reported post [\(newTitle)](//stackoverflow.com/q/\(questionID!))"
-            
-            message.room.postMessage(messagePost)
-        }
+        var filterResult = reporter.checkPost(question)
+        
+        filterResult [0] = FilterResult (type: .manuallyReported, header: "Manually reported question", details: "Question manually reported.")
```

Comment:
How about `filterResult.append(...)` so that we don't clobber the Bayes stats?

Also, the details should be like a `why` string.  How about something like `Post manually reported by \(message.user): https://chat.\(message.room.client.host.rawValue)/transcript/message.id#message.id`?